src/snippets/title-header.liquid

```diff
@@ -1 +1 @@
-<h1 class="title-header sans-serif-uppercase-l text-center">{{ title }}</h1>
\ No newline at end of file
+<h1 class="title-header sans-serif-uppercase-l text-center {% unless show_on_desktop %}large-up--hide{% endunless %}">{{ title }}</h1>
```

Comment:
@sarahscott i'm self-merging this pr, so if you rebase on master you can use this snippet in collection view + pages to include the page title with consistent spacing on every page. show_on_desktop param toggles whether or not it is mobile-only